packages/api-reference/index.html

```diff
@@ -20,14 +20,15 @@
     <script type="module">
       Scalar.createApiReference('#app', {
         sources: [
-          {
-            url: 'https://petstore.swagger.io/v2/swagger.json',
-          },
           {
             title: 'Scalar Galaxy', // optional, would fallback to 'API #1'
             slug: 'scalar-galaxy', // optional, would be auto-generated from the title or the index
             url: 'https://cdn.jsdelivr.net/npm/@scalar/galaxy/dist/latest.json',
           },
+          {
+            title: 'Swagger Petstore',
+            url: 'https://petstore.swagger.io/v2/swagger.json',
+          },
```

Comment:
lol I did this in so many branches already, but it never made it to main

Reply:
I swear I've done it before too 😂